src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp

```diff
@@ -515,7 +515,50 @@ void jit_floor_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const s
 std::set<std::vector<element::Type>> jit_floor_emitter::get_supported_precisions(const std::shared_ptr<ov::Node>& node) {
     return {{element::f32}};
 }
+/// FLOOR_MOD ///
+jit_floor_mod_emitter::jit_floor_mod_emitter(dnnl::impl::cpu::aarch64::jit_generator *host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const std::shared_ptr<ov::Node>& node)
+    : jit_emitter(host, host_isa, node, get_arithmetic_binary_exec_precision(node)) {
+}
+
+jit_floor_mod_emitter::jit_floor_mod_emitter(dnnl::impl::cpu::aarch64::jit_generator *host,
+                                             dnnl::impl::cpu::aarch64::cpu_isa_t host_isa,
+                                             const ov::element::Type exec_prc): jit_emitter(host, host_isa, exec_prc) {
+}
+
+size_t jit_floor_mod_emitter::get_inputs_count() const { return 2; }
+
+size_t jit_floor_mod_emitter::get_aux_vecs_count() const { return 1; }
 
+void jit_floor_mod_emitter::emit_impl(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    if (host_isa_ == dnnl::impl::cpu::aarch64::asimd) {
+        emit_isa<dnnl::impl::cpu::aarch64::asimd>(in_vec_idxs, out_vec_idxs);
+    } else {
+        OV_CPU_JIT_EMITTER_THROW("Can't create jit eltwise kernel");
+    }
+}
+
+template <dnnl::impl::cpu::aarch64::cpu_isa_t isa>
+void jit_floor_mod_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    OV_CPU_JIT_EMITTER_ASSERT(exec_prc_ == ov::element::f32, "unsupported precision: " + exec_prc_.to_string());
+
+    using TReg = typename dnnl::impl::cpu::aarch64::cpu_isa_traits<isa>::TReg;
+
+    TReg dividend = TReg(in_vec_idxs[0]);
+    TReg divisor = TReg(in_vec_idxs[1]);
+    TReg r = TReg(out_vec_idxs[0]);
+    TReg aux = TReg(aux_vec_idxs[0]);
+
+    h->fdiv(aux.s, dividend.s, divisor.s);
+    h->frintm(aux.s, aux.s);
+    h->fmul(aux.s, aux.s, divisor.s);
+    h->fsub(r.s, dividend.s, aux.s);
```

Comment:
>It makes more sense to use r instead of the aux register

Yes, this is what I meant 👍🏼 

> The mod function in same file also follows the aux approach should i make changes there too?

Now we don't need to `aux_vec_idx`, so the method `jit_floor_mod_emitter::get_aux_vecs_count()` should return `0`.

Reply:
so instead of removing this line i just need to return 0?
```.cpp
size_t jit_floor_mod_emitter::get_aux_vecs_count() const { return 0; }
```
### Also am i allowed to make same changes in MOD functionality too? because i see that they follow same  aux approach!

Reply:
By the way, I have found the PR https://github.com/openvinotoolkit/openvino/pull/25701 which replaces `r` with `aux`  register in `jit_mod_emitter`. Probably, the using `r` leaded to accuracy problems (maybe some instructions on some platforms doesn't support work with the same src and dst registers). Because of that I recommend to leave it so (with `aux` register).

Now, please see the comment https://github.com/openvinotoolkit/openvino/pull/27706#pullrequestreview-2457157685 - need to support CPU-specific tests for this op.